actions/registry.go

```diff
@@ -145,6 +145,27 @@ func Unpin(node *p2p.QriNode, ref repo.DatasetRef) (err error) {
 	}
 
 	return reg.Unpin(ref.Path, pk)
+
+}
+
+// RegistryList gets a list of the published datasets of a repo's specific registry
+func RegistryList(node *p2p.QriNode, limit, offset int) (datasets []*repo.DatasetRef, err error) {
```

Comment:
Could you swap the order of the `limit` and `offset` parameters? Everywhere else (nearly?) in the codebase uses the order of `offset` then `limit`, and since both are int's, we can't use the compiler to make sure the right values are passed in, we have to stick to a consistent convention.

Reply:
I'm not seeing any cases in where `offset` is before `limit`. I was following the convention that was set in `lib.List`, `actions.listDatasets`, `base.libDatasets` basically our local dataset listing.

Totally agree it should be uniform!

Reply:
My mistake, you're totally right. Nevermind then!